### PR TITLE
Move CommonPaths to Shared

### PR DESCRIPTION
**Pull request by cinqmilleans**
Move CommonPaths to Shared.

- This pr moves CommonPaths to Shared. It is necessary to move the storage to the backend soon.
- The CommonPaths file has been restyled and its dictionary has become readonly.
- A cleanup has been done in the using files that depend on CommonPaths.
- This pr also moves the localization service and extension from Backend to Shared because CommonPaths uses it. This is its most natural place.

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

File: src/Files.Shared/Extensions/LocalizationExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.Shared.Services;
+
+namespace Files.Shared.Extensions
+{
+    public static class LocalizationExtensions
```
Comment (d2dyno1) [inline src/Files.Shared/Extensions/LocalizationExtensions.cs:6]: That's not supposed to be here 🤔

Comment (cinqmilleans) [reply src/Files.Shared/Extensions/LocalizationExtensions.cs:6]: Sorry, I don't understand your comment.

Comment (d2dyno1) [reply src/Files.Shared/Extensions/LocalizationExtensions.cs:6]: I mean, we decided that it should stay in Files.Backend

Comment (cinqmilleans) [reply src/Files.Shared/Extensions/LocalizationExtensions.cs:6]: An alternative solution would be to place CommonPaths in Backend. It's more logical.

**Comment by cinqmilleans** [comment]
@d2dyno1 I open a new pr to move CommonPaths to Backend. This is therefore closed.

**Pull request closed without merging**